client/src/index.html

```diff
@@ -104,6 +104,21 @@
     </script>
     <script type="module" src="/build/app.esm.js"></script>
     <script nomodule src="/build/app.js"></script>
+    <!-- prioritize assets -->
+    <link rel="modulepreload" href="/build/app.esm.js" />
+    <link rel="modulepreload" href="/build/docs-router.entry.js" />
```

Comment:
Aren't these `href` going to be invalid due to the hashing of entries that stencil does when it bundles?

![Screen Shot 2020-06-15 at 10 50 13 AM](https://user-images.githubusercontent.com/3868826/84689563-035f8180-aef6-11ea-95a3-398caf8c0c61.png)

Reply:
Nope. Those are other modules. The ones listed in the preloads will keep their names.